app/models/candidate.rb

```diff
@@ -1,4 +1,6 @@
 class Candidate < ActiveRecord::Base
+  has_many :answers
```

Comment:
What do you think about adding `dependent: :destroy` here, so if a candidate is deleted their answers go along with them?